File: 3 cource/2 sem/Software quality and testing/Lab 1/main.cpp
#include <iostream>
#include <cmath>

float GetRowSum(float x)
{
    float sum = 0;
    int d = 1;

    for(int i = 0; i < 15; i++)
    {
        sum += 1 / (d * pow(x, d));
        d += 2;
    }

    return sum;
}

int main() {
    while(true)
    {
        float x = 0;
        std::cout << "Enter X:";
        std::cin >> x;

        if(std::abs(x) < 1 || std::abs(x) > 3) {break;}

        std::cout << "Row sum: " << GetRowSum(x) << std::endl;
    }
    return 0;
}

File: 3 cource/2 sem/Software quality and testing/Lab 1/main.py
while True:
    x = float(input("Enter X:"))

    if abs(x) < 1 or abs(x) > 3: break

    summ = 0
    d = 1

    for i in range(15):
        summ += 1 / (d * pow(x, d))
        d += 2

    print("Row sum: " + str(summ))
